packages/waku/src/lib/renderers/rsc-renderer.ts

```diff
@@ -188,7 +198,46 @@ export async function renderRsc(
     ) {
       // XXX This doesn't support streaming unlike busboy
       const formData = parseFormData(bodyStr, contentType);
-      args = await decodeReply(formData);
```

Comment:
There are something wrong, formData could include server references which need webpack runtime @dai-shi

Reply:
https://github.com/facebook/react/blob/ddcecbbebf6af3fa32a323c0591dad1f63587aeb/packages/react-server/src/ReactFlightReplyServer.js#L927-L941

Reply:
Alright. It's reasonable that it requires the second argument `moduleMap`.